src/app/components/mno-apps/mno-app-listing.coffee

```diff
@@ -23,12 +24,36 @@ angular.module 'mnoEnterpriseAngular'
       #====================================
       # Scope Management
       #====================================
+      vm.linkFor = (app) ->
+        "#/marketplace/#{app.id}"
 
-      vm.appsFilter = (app) ->
-        if (vm.searchTerm? && vm.searchTerm.length > 0) || !vm.selectedCategory
-          return true
-        else
-          return _.contains(app.categories, vm.selectedCategory)
+      vm.onSearchChange = () ->
+        vm.selectedCategory = ''
```

Comment:
I'm unsure why you're reseting `vm.selectedCategory` to `''`?
You're resetting it again on L52.
Just want to make sure this doesn't have any unintended effects.